client/consensus/src/collators/lookahead.rs

```diff
@@ -482,6 +481,7 @@ where
             select! {
                 maybe_relay_parent_header = import_notifications.next() => {
                     if maybe_relay_parent_header.is_none() {
+                        log::info!("breaking");
```

Comment:
```suggestion
```

Reply:
fixed in [82242eb](https://github.com/moondance-labs/tanssi/pull/662/commits/82242eb5310be2e51fad9bb311e231f63c8199bc)